src/config.js

```diff
@@ -145,12 +145,12 @@ export function loadJSConfigFile(filePath) {
 }
 
 export async function discoverConfigFiles({ getHomeDir = os.homedir } = {}) {
-  const magicConfigName = 'web-ext-config.js';
+  const magicConfigName = '.web-ext-config.js';
 
   // Config files will be loaded in this order.
   const possibleConfigs = [
     // Look for a magic hidden config (preceded by dot) in home dir.
-    path.join(getHomeDir(), `.${magicConfigName}`),
+    path.join(getHomeDir(), magicConfigName),
```

Comment:
**requested changes**: Personally I'd prefer if we would add `.web-ext-config.js` as an additional config file we look for as part of the discovery, in addition to `web-ext-config.js`, and it seems that was also what @Rob--W was proposing in https://github.com/mozilla/web-ext/issues/2812#issuecomment-1643919085

Would you mind to tweak this PR to do that? (looking for `.web-ext-config.js` in the home and in the project dir right after the one named `web-ext-config.js`.

-----

**thought:** I'm wondering if we should also add a warning message when loading for the files named `web-ext-config.js` (the ones without the leading dot) so that we can consider to deprecate them in the future, but I would keep that out of the PR for now, we can eventually consider to emit warnings in a separate followup PR.

Reply:
Do you mean to use `web-ext-config.js` and `.web-ext-config.js`? I will tweak it.